src/include/network/abstract_command_factory.h

```diff
@@ -1,24 +0,0 @@
-#pragma once
-#include <memory>
-#include "network/postgres/postgres_network_commands.h"
-
-#define MAKE_COMMAND(type) std::static_pointer_cast<AbstractNetworkCommand, type>(std::make_shared<type>(packet))
-
-namespace terrier::network {
-
-/**
- * A command factory takes in input packets and converts them to commands usable by the protocol interpreters
- */
-class AbstractCommandFactory {
- public:
-  /**
-   * Convert a packet to a command.
-   * @param packet the input packet
-   * @return a shared_ptr to the converted command
-   */
-  virtual std::shared_ptr<AbstractNetworkCommand> PacketToCommand(InputPacket *packet) = 0;
```

Comment:
From the PR description:

> I deleted abstract_command_factory as I don't see anything being shared between ITP and Postgres.

Both `PostgresCommandFactory` and `ITPCommandFactory` can share the `PacketToCommand` method. Since `PostgresNetworkCommand` and `ITPNetworkCommand` both inherit from `AbstractNetworkCommand`, both of the derived methods can have the same return type

Reply:
I see; I have added this class back.